backend/src/routes/preferences.js

```diff
@@ -1,6 +1,23 @@
 import express from 'express';
+import Preference from '../mongo/models/Preference';
 const router = express.Router();
 
-// TODO: Handlers...
+// POST: Creating a preferences document in the preferences collection
+router.post('/', async (req, res) => {
```

Comment:
Your api contract says the path should be '/preferences'. Which one are you using?

Reply:
The path is defined in index.js:
<img width="390" alt="Screen Shot 2021-03-16 at 8 29 35 pm" src="https://user-images.githubusercontent.com/43579489/111271630-56897f00-8696-11eb-8b4d-80024bd233a5.png">

Reply:
oops my bad